src/index.css

```
@@ -1,41 +1,35 @@
-/* Font settings */
+@import 'basic-css-reset';
```

```suggestion
/**
 * Set all of the layers to be used by CSS.
 * It allows us to define styles that can be overriden by the next step without using !important everywhere.
 * And not have to be fighting the cascade.
 */
@layer reset, fonts, design-system, base, pages, components, theme, print, accessibility;

/**
 * Import the reset into the reset layer.
 */
@import './basic-css-reset.css' layer(reset);
```

The import is a URL, so I prefer having it as a relative URL here (starting with a `./`).

Also added [`@layer`](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer) so we can organize things into layers.

Here is a video on CSS layers: https://www.youtube.com/watch?v=ilrPpSQJb3U